becca-new-app/src/App.js

```diff
@@ -41,10 +58,12 @@ async function fetchData(setTodoList, setIsLoading) {
 function App() {
 	const [todoList, setTodoList] = useState([]);
 	const [isLoading, setIsLoading] = useState(true);
+	const [sortOrder, setSortOrder] = useState("asc");
```

Comment:
What do these do? It doesn't seem like they're being used by the actual `fetchData` function